components/org.wso2.carbon.micro.integrator.initializer/src/main/java/org/wso2/carbon/micro/integrator/initializer/ServiceBusInitializer.java

```diff
@@ -217,7 +218,7 @@ protected void deactivate(ComponentContext ctxt) {
 
     private void initPersistence(SynapseConfigurationService synCfgSvc, String configName) throws AxisFault {
         // Initialize the mediation persistence manager if required
-        ServerConfiguration serverConf = ServerConfiguration.getInstance();
+        CarbonServerConfigurationService serverConf = CarbonServerConfigurationService.getInstance();
```

Comment:
Can we not use Carbon word here?

Reply:
and is it an osgi service?

Reply:
Used as we are reading configs from carbon.xml

Reply:
> is it an osgi service?

yes